commit 19000fad573138b2b0576c093237c4a708b76020
Author: Neha Ojha <nojha@redhat.com>
Date:   Thu Dec 3 20:34:53 2020 +0000

    mgr: don't add config values to mgr log in config_callback and module_config
    
    The original code has been commented out and left for future debugging
    purposes.
    
    Signed-off-by: Neha Ojha <nojha@redhat.com>

diff --git a/src/mgr/PyModuleRegistry.cc b/src/mgr/PyModuleRegistry.cc
index ad461aa47bc..221b93beed7 100644
--- a/src/mgr/PyModuleRegistry.cc
+++ b/src/mgr/PyModuleRegistry.cc
@@ -431,7 +431,10 @@ void PyModuleRegistry::handle_config(const std::string &k, const std::string &v)
   std::lock_guard l(module_config.lock);
 
   if (!v.empty()) {
-    dout(10) << "Loaded module_config entry " << k << ":" << v << dendl;
+    // removing value to hide sensitive data going into mgr logs
+    // leaving this for debugging purposes
+    // dout(10) << "Loaded module_config entry " << k << ":" << v << dendl;
+    dout(10) << "Loaded module_config entry " << k << ":" << dendl;
     module_config.config[k] = v;
   } else {
     module_config.config.erase(k);
